art.cls

```diff
@@ -170,15 +120,58 @@
 \newcommand{\absfont}{\normalfont\fontsize{10pt}{11pt}\selectfont}
 \newcommand{\keywordsfont}{\normalfont\fontsize{8pt}{11pt}\selectfont}
 \newcommand{\footnotefont}{\normalfont\fontsize{7pt}{9pt}\selectfont}
+\newcommand{\monofontscale}{0.85}
 }
 
-% AMS maths packages
-\RequirePackage{
-  amsmath,
-  amsfonts,
-  amsthm,
-  % amssymb
-}
+\usepackage{iftex}
+\ifPDFTeX
+  \RequirePackage[LGR,TS1,T1]{fontenc}
+  \RequirePackage[utf8]{inputenc}
+  \RequirePackage{cmap}
+  \RequirePackage{textcomp}
+\else
+  % \usepackage[euler]{fontsetup}
+  % \usepackage[concrete]{fontsetup}
+  % \usepackage[libertinus]{fontsetup}
+  % \usepackage[kerkis]{fontsetup}
+  % \usepackage[gfsdidot]{fontsetup}
+
+  % \usepackage{sansmathfonts}
+  % \renewcommand*\familydefault{\sfdefault}
+
+  % \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
+
+  % \defaultfontfeatures{Scale=MatchLowercase}
+  % \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
+\fi
+
+\RequirePackage[libertine,amsthm]{newtx}
+
+\RequirePackage[varqu]{zi4} % inconsolata
+\ifPDFTeX\else
```

Comment:
if we change this line to
```
\ifPDFTeX\RequirePackage[scaled=.825]{GoMono}\else
```
I think we would get something that looks about right if one insists in pdflatex for some reason, but  needs minted listings
![image](https://github.com/anoma/art-template/assets/13623940/7058a980-aaa9-4be3-896e-fa5cedb560ed)